lib/raven/processor/sanitizedata.rb

```diff
@@ -5,6 +5,7 @@ class Processor::SanitizeData < Processor
     INT_MASK = 0
     DEFAULT_FIELDS = %w(authorization password passwd secret ssn social(.*)?sec)
     CREDIT_CARD_RE = /^(?:\d[ -]*?){13,16}$/
+    REGEX_SPECIAL_CHARACTERS = %w(. $ ^ { [ ( | ) * + ?)
```

Comment:
We want to strictly match on whole words such as "name" and "foo_bar", which do not include `REGEX_SPECIAL_CHARACTERS `, in `sanitize_fields`. So, if we allow regexes in both, we would still have the same check?